.github/PULL_REQUEST_TEMPLATE/module_update.md

```diff
@@ -0,0 +1,64 @@
+<!--
+This project's monthly turnover might well be beyond 1500 PRs. Since we are community driven,
+let's do tit for tat: submitters are kindly ask to pick an open pull request for review.
+
+Start with packages updates. They are relatively simple.
+Since you'll be required to do those very steps below, it'll be easy.
+
+Marvin needs_reviewer: https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+label%3Aneeds_reviewer+
+Marvin needs_merger:   https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+label%3Aneeds_merger+
+Reviewing guidelines:  https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#chap-reviewing-contributions
+-->
+
+##### Motivation for this change
+
+
+##### General review
+
+- [ ] changes are backward compatible
+- [ ] removed options are declared with `mkRemovedOptionModule`
+- [ ] changes that are not backward compatible are documented in release notes
+- [ ] module tests succeed on ARCHITECTURE
+- [ ] options types are appropriate
+- [ ] options description is set
```

Comment:
This would raise at least a warning during eval IIRC.

Reply:
I wasn't concerned at all with correctness of those items sine they are verbatim from the review guidelines. If those guidelines are updated, so should those commit templates. Or even guidelines could refrain from specifying those options and graduate into explicitly referencing those checklists. Something for a follow up PR though.